pkg/machine/qemu/options_darwin.go

```diff
@@ -11,3 +12,15 @@ func getRuntimeDir() (string, error) {
 	}
 	return tmpDir, nil
 }
+
+// makeSymlink for macOS creates a symlink in $HOME/.podman/
+// for a machinefile like a socket
+func (m *MachineFile) makeSymlink(symlink *string) error {
+	homedir, err := os.UserHomeDir()
```

Comment:
before doing this would it make sense to check if path > 104 and skip it otherwise?